openslides/users/static/js/users/site.js

```diff
@@ -312,26 +322,6 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
                     }
                 },
                 {
-                    key: 'default_password',
```

Comment:
If this field is remove, how can an admin reset or change the password of another user?